app/controllers/travel_advice_controller.rb

```diff
@@ -37,7 +37,7 @@ def show
   # TODO: Controllers should provide a presenter or a publication.
   # These objects duplicate roles (see `presenter || @publication`) in views.
   def publication
-    content_item if country_page?
```

Comment:
I'm still very suspicious about a publication method that returns a content_item, but I think that's a fix outside of the scope of this change.